source/github-helpers/get-default-branch.ts

```diff
@@ -14,7 +14,11 @@ const branchInfoRegex = /([^ ]+)\.$/;
 
 export default cache.function(async (repository: Partial<RepositoryInfo> = getRepositoryInfo()): Promise<string> => {
 	if (JSON.stringify(repository) === JSON.stringify(getRepositoryInfo())) {
-		if (!isForkedRepo()) {
+		if (pageDetect.isRepoRoot() && !String(getRepoPath()).startsWith('tree/')) {
```

Comment:
If you want the absolute repo root, then just look for it. This should be it:
```suggestion
		if (getRepoPath() === '') {
```

Reply:
I'd call that `repo home`

Reply:
Will send a PR now